.github/workflows/centos_7.yml

```diff
@@ -24,6 +24,12 @@ jobs:
       fail-fast: false
 
     steps:
+      # Permissions correction is needed for self-host runners,
+      # where work path is saved between different workflows runs.
+      - name: correct permissions in working directory
+        shell: bash
+        run: |
+          sudo chown -R $(id -u):$(id -g) .
```

Comment:
Can we avoid code duplication? I see that the 'environment' action goes after the checkout action (at least in some jobs and hopefully can go after it in all jobs), so maybe place it here?

Reply:
Ah, I see, we need to do it before checkout. How about some common cleanup action? Or moving the environment action before the checkout one?

Reply:
I see, PR #5954 requires 'environment' be after checkout.

Anyway, we can have one pre-checkout and one post-checkout common action. (What do you think about names?)

Reply:
Actually the main issue that we don't have actions available till checkout actions run - it creates needed actions files.

Reply:
Okay, got it.